users/tests/test_create_teams.py

```diff
@@ -16,9 +16,10 @@ def test_create_team_without_title(self):
 
     @pytest.mark.django_db(transaction=True)
     def test_create_team_with_same_name(self):
-        assert len(Team.objects.all()) == 0
+        length = len(Team.objects.all())
```

Comment:
"length" is a too generic name
maybe it will be more readable like:
```suggestion
        base_team_len = len(Team.objects.all())
```